content/localized/ja/UserGuide-chapter1.adoc

```diff
@@ -1,393 +0,0 @@
-= Chapter 1. OptaPlanner イントロダクション
```

Comment:
This Japanese user guide chapter 1 was hopelessly stale (Still pointing to version 6.4 images). It's better to not have it all.